### PR TITLE
width fixed: edit feature form

### PR DESCRIPTION
**Pull request by juFelipe**
https://github.com/TeselaGen/hde/issues/1818
props inLine removed, that fix the problem

### CODE REVIEW

File: src/Editor/index.js [hunk 1]
```diff
@@ -904,6 +904,7 @@ export class Editor extends React.Component {
             className={"ve-clickable ve-close-panel-button"}
             style={{
               zIndex: 15001,
+              minWidth: "200px !important",
```
Comment (tnrich) [inline src/Editor/index.js]: @juFelipe what's this fixing here? Seems like it would make the close button unnecessarily large?

Comment (juFelipe) [reply src/Editor/index.js]: oh I was trying to fix another bug, looking for a solution and I forget to delete that, I'm sorry

Comment (tnrich) [reply src/Editor/index.js]: no worries. change it and I'll merge it in and publish :)

**Comment by tnrich** [comment]
I've addressed these issues in a different way. Thanks @juFelipe

**Pull request closed without merging**